### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'nodeType'

### ISSUE THREAD

**Issue by arthuragone** [nextgis/quickmapservices#146]
Hello, When trying to use the plugin I always got this error:

>   File "/Users/arthur/.qgis2/python/plugins/quick_map_services/quick_map_services.py", line 172, in insert_layer
			    add_layer_to_map(ds)
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/qgis_map_helpers.py", line 142, in add_layer_to_map
			    if selected_node.nodeType() == selected_node.NodeGroup:
			AttributeError: 'NoneType' object has no attribute 'nodeType'

**Comment by simgislab** [comment]
please provide the following:
version of QGIS
version of QMS
type and version of OS

what exactly are you doing before getting this error

**Comment by arthuragone** [comment]
QGIS 2.18.13
QuickMap Services 0.19.3

OS: OsX

Doing nothing before getting this error, just trying to add any web raster (BING, Google, OSM, etc.)

**Comment by arthuragone** [comment]
I completely changed environnement but got again this bug:

```
  File "C:/Users/AVIGNACQ/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\quick_map_services\quick_map_services.py", line 176, in insert_layer
              add_layer_to_map(ds)
              File "C:/Users/AVIGNACQ/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\quick_map_services\qgis_map_helpers.py", line 148, in add_layer_to_map
              if selected_node.nodeType() == selected_node.NodeGroup:
             AttributeError: 'NoneType' object has no attribute 'nodeType'
```

Windows 10 PRO 64 bits
QGIS 3.4.2

**Comment by AlexRiggs** [comment]
I have also run into this issue. 

QGIS 3.8.2-Zanzibar
QMS 0.19.10.1
OS Linux Ubuntu 18.04.3

**Comment by AlexRiggs** [comment]
I was able to use the plugin after rebooting then running `sudo apt update && sudo apt -y upgrade`

**Comment by 514ckw4r3** [comment]
Fixed in 0.19.19
